owasp-top10-2017-apps/a8/amarelo-designs/app/app.py

```diff
@@ -19,11 +22,20 @@ def login():
     
         if username == "admin" and password == "admin":
```

Comment:
What do you think about chaning this default password here? Using an environment variable as you did with `JWT_SECRET_KEY` would be great

Reply:
Thanks, @gustavocovas. Now the application is getting the user's credentials from the environment variables.